backend/Dockerfile.ops-api

```diff
@@ -1,26 +1,30 @@
-FROM python:3.11-slim
-# hadolint ignore=DL3008
-RUN apt-get update && apt-get -y install curl --no-install-recommends && apt-get clean && rm -rf /var/lib/apt/lists/*
-RUN pip install --no-cache-dir --upgrade pip==23.0.0 pipenv==2023.2.4 && useradd -ms /bin/bash app
+FROM alpine:3
+
+ENV PYTHONUNBUFFERED=1
+ENV PYTHONPATH=/home/app
+
+RUN apk update && apk add --no-cache curl=8.7.1-r0 python3=3.12.3-r1 py3-pip=24.0-r2
```

Comment:
wonder if renovate will keep those versions up to date.....

Reply:
I added some config to attempt to accomplish this

Reply:
cool